### DISCUSSION TITLE
Force-Feedback Plugin support

### DISCUSSION THREAD

**Discussion by cangarujack** [emoose/OutRun2006Tweaks#231] [Suggestions]
### Describe the feature you'd like to see added

It is possible?

**Comment by emoose** [comment]
It might be up to the FFB plugin devs to decide, I know https://github.com/Boomslangnz/FFBArcadePlugin has support for 2SP TeknoParrot, would guess a lot of that code could probably be ported to C2C fine.

There is FXT which afaik adds FFB effects to C2C (and I think the dev Howard did work on the OR2 support in that arcade plugin), I'd heard some people had FXT working with Tweaks a while ago, but there's been a lot of new tweaks added since that so not sure if it's still the case.

**Comment by Zoukambada** [comment]
A [feature request](https://github.com/Boomslangnz/FFBArcadePlugin/issues/84) was done by @cangarujack in FFBArcadePlugin GitHub already.

**Comment by Zoukambada** [comment]
@emoose do you know Darthsol (aka S0L) from Gamefaqs Outrun2006 Message boards?
S0L worked on Sumo Digital and was very active on these forums in the past.
That message board is a gold mine of curiosities, differences and facts about all Outrun2 ports because of they answering questions from other players.

This topic below talks about the Force Feedback. S0L says that the OutRunSP2 from Japan has it implemented (something that isn't available in the US and Europe):

https://gamefaqs.gamespot.com/boards/930972-outrun-2006-coast-2-coast/48505940

"We're supporting certain Logitech wheels on the PS3 version of OutRun Online Arcade, can't remember which ones off the top of my head though. I think it pretty much works with any of them that are designed for PS3 specifically.

We've also got support in OOA for the 360 wheel too.

Japanese PS2 version (OutRun2SP) also support force feedback on Logitech Wheels too. Note that's specific to the Japanese PS2 version only though, the PAL and US versions don't support it (as you have force feedback or online - well if you want it at 60fps and not 30fps)."

Maybe S0L can help you with more information in any topic about OR2006.

**Reply by d-b-c-e** [reply]
Might not help any but I've been working on my own as well

https://github.com/d-b-c-e/OutRun2006Tweaks-FFB

**Comment by Zoukambada** [comment]
@cangarujack and @emoose , there are currently two working in progress Tweaks forks with FFB feature:

One from @hyp36rmax
https://github.com/hyp36rmax/multi-device-input/releases/latest

And another from @thp32tt
https://github.com/thp32tt/OutRun2006Tweaks/releases

Both devs are active on Tweaks Discord server at the channel #for-wheel-players and people are beta testing and giving feadback there.
